testdata/test_exit_code.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```

Comment:
Add a `set -e` so this script exits on any error. May seem silly for such a tiny script, but you never know if it'll grow in the future.